src/NUnitEngine/nunit.engine/Internal/ResultHelper.cs

```diff
@@ -46,6 +45,10 @@ public static class ResultHelper
         /// Aggregate the XmlNodes under a TestEngineResult into a single XmlNode.
         /// </summary>
         /// <param name="result">A new TestEngineResult with xml nodes for each assembly or project</param>
+        /// <param name="elementName"></param>
```

Comment:
I agree.  There is no reason to add the missing parameters to the XML docs unless they are added with comments that provide useful information.